site/class-1-bar/vriesm060/03 - Bar Chart/data.tsv

```diff
@@ -0,0 +1,27 @@
+letter	frequency
```

Comment:
You’ve uploaded your files to `site/class-1-bar/vriesm060/03 - Bar Chart` instead of `site/class-1-bar/vriesm060`!